### ISSUE TITLE
Pending user invites aren't appearing automatically in the users list

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#2276]
### Description

- if you invite someone, then click `back to all users`, the pending invite doesn’t pop up.
- If you click on some other tab, then click back to `user management`, the pending invite appears

**Comment by rephus** [comment]
@TuringLovesDeathMetal it seems this has been fixed already, could you confirm ? Tested on version `- v0.126.0`
 ![Uploading Peek 2022-05-31 08-53.gif…]()

**Comment by rephus** [comment]
Yeah ,you may be right, let's wait until https://github.com/lightdash/lightdash/pull/2282 is done and test this again.

**Comment by rephus** [comment]
Tested again on analytics, looks good to me 
![Peek 2022-06-07 08-56](https://user-images.githubusercontent.com/1983672/172315660-3fc0fbf6-5149-498b-953a-d7eb1420ba3a.gif)

**Comment by ZeRego** [comment]
Closing this ticket since we can't reproduce. I also remember fixing and testing this when working on the invites 🤔 We should start posting the app version on bugs so it's easier to find out if a fix has already been shipped or not.
